pipeline-modules/infra-service/azure/terraform.yaml

```diff
@@ -82,7 +82,7 @@ inputs:
         - WARN
         - ERROR
         - JSON
-      default: DEBUG
+      default: ERROR
```

Comment:
The other clouds default this to DEBUG, I think we should try to keep the modules as consistent as possible.